### ISSUE TITLE
SetNoCollideWithTeammates only works with teams 1-4

### ISSUE THREAD

**Issue by TIMONz1535** [Facepunch/garrysmod-issues#4878]
### Details

SetNoCollideWithTeammates works correctly only with team index 1-4. And does not work at all if the team index is >4.

If you have a team 1-4 and another player has an index >4 then he can pass through your collision. At this moment you will stuck in him.

### Steps to reproduce

Set to players a team with index 5 and enable SetNoCollideWithTeammates, players will not be able to pass through each other.
If there are players with index 1-4, then players with index 5 will be able to pass through all of them when the first ones get stuck.

**Comment by FlorianLeChat** [comment]
Isn't fixed here since #2757 ?

**Comment by robotboy655** [comment]
No, it's a limitation that cannot be lifted unfortunately, as it relies on CONTENTS_TEAM1-4 enums which cannot be infinite.

**Comment by TIMONz1535** [comment]
Ok. If it can't be fixed please document this on wiki as "WARNING" for these things
- team.SetUp
- Player:SetTeam
- Player:SetNoCollideWithTeammates (also shared)
